src/main.zig

```diff
@@ -2397,6 +2398,8 @@ fn buildOutputType(
                 } else if (mem.eql(u8, arg, "--version")) {
                     try std.io.getStdOut().writeAll("zig ld " ++ build_options.version ++ "\n");
                     process.exit(0);
+                } else if (mem.startsWith(u8, arg, "-Map=")) {
+                    linker_map_file = arg["-Map=".len..];
```

Comment:
This handles `-Map=file` which is enough for me. Should `-Map file` be handled as well?